roles/kubeadm-master/tasks/main.yaml

```diff
@@ -44,6 +44,24 @@
     block: "    - --authentication-token-webhook-config-file=/etc/kubernetes/pki/webhook.kubeconfig.yaml"
     path: /etc/kubernetes/manifests/kube-apiserver.yaml
 
+- name: In case of upgrade make sure container versions are right for kube-apiserver
```

Comment:
We should probably handle upgrades with `kubeadm upgrade` now it's [supported upstream](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-upgrade/).

I can merge this for now, but we'll need to figure out a cleaner upgrade path and probably stop managing these files with ansible.